cmd/init.go

```diff
@@ -0,0 +1,163 @@
+  //
+  // cloud: {
+  //   // The ID of the project to which the test is assigned in the k6 Cloud UI.
+  //   // By default tests are executed in default project.
+  //   projectID: "",
+  //   // The name of the test in the k6 Cloud UI.
+  //   // Test runs with the same name will be grouped.
+  //   name: "{{ .ScriptName }}"
+  // },
+
+  // Uncomment this section to enable the use of Browser API in your tests.
+  //
+  // See https://k6.io/docs/using-k6-browser/running-browser-tests/ to learn more
+  // about using Browser API in your test scripts.
+  //
+  // scenarios: {
+  //   // The scenario name appears in the result summary, tags, and so on.
+  //   SCENARIO NAME: {
+  //     // Mandatory parameter for browser-based tests.
+  //     executor: 'shared-iterations',
+  //     options: {
+  //       browser: {
+  //         // This is a mandatory parameter that instructs k6 to launch and
+  //         // connect to a chromium-based browser, and use it to run UI-based
+  //         // tests.
+  //         type: 'chromium',
+  //       },
+  //     },
+  //   },
+  // }
+};
+
+// The function that defines VU logic.
+//
+// See https://k6.io/docs/examples/tutorials/get-started-with-k6/ to learn more
+// about authoring k6 scripts.
+//
+export default function() {
+  http.get('https://test.k6.io');
+  sleep(1);
+}
+`))
+
+type initScriptTemplateArgs struct {
+	ScriptName string
+}
+
+// initCmd represents the `k6 init` command
+type initCmd struct {
+	gs             *state.GlobalState
+	overwriteFiles bool
+}
+
+func (c *initCmd) flagSet() *pflag.FlagSet {
+	flags := pflag.NewFlagSet("", pflag.ContinueOnError)
+	flags.SortFlags = false
+	flags.BoolVarP(&c.overwriteFiles, "force", "f", false, "Overwrite existing files")
+
+	return flags
+}
+
+func (c *initCmd) run(cmd *cobra.Command, args []string) error { //nolint:revive
+	target := defaultNewScriptName
+	if len(args) > 0 {
+		target = args[0]
+	}
+
+	fileExists, err := fsext.Exists(c.gs.FS, target)
+	if err != nil {
+		return err
+	}
+
+	if fileExists && !c.overwriteFiles {
+		return fmt.Errorf("%s already exists, please use the `--force` flag if you want overwrite it", target)
+	}
+
+	fd, err := c.gs.FS.Create(target)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		_ = fd.Close() // we may think to check the error and log
+	}()
+
+	if err := defaultNewScriptTemplate.Execute(fd, initScriptTemplateArgs{
+		ScriptName: path.Base(target),
+	}); err != nil {
+		return err
+	}
+
+	valueColor := getColor(c.gs.Flags.NoColor || !c.gs.Stdout.IsTTY, color.Bold)
+	printToStdout(c.gs, fmt.Sprintf(
+		"Initialized a new k6 test script in %s. You can now execute it by running `%s run %s`.\n",
+		valueColor.Sprint(target),
+		c.gs.BinaryName,
+		target,
+	))
+
+	return nil
+}
+
+func getCmdInit(gs *state.GlobalState) *cobra.Command {
+	c := &initCmd{gs: gs}
+
+	exampleText := getExampleText(gs, `
+  # Create minimal k6 script.js in the current directory
```

Comment:
Just aligning with the other help, where mostly `k6 script` mentioned

```suggestion
  # Create a minimal k6 script in the current directory. By default, k6 creates script.js.
```